pgx-utils/src/lib.rs

```diff
@@ -517,7 +517,8 @@ pub fn anonymonize_lifetimes(value: &mut syn::Type) {
                             match arg {
                                 // rename lifetimes to the anonymous lifetime
```

Comment:
comment no longer matches the code below.  Why are we now using `'static`?

Reply:
Ah, thanks!

We need anything using a lifetime to use the same lifetime so that the `TypeId`s match (`&'a Foo` and `&'b Foo` have different `TypeId`s)